project-extender/src/providers/jira-server-provider/JiraServerProvider.ts

```diff
@@ -34,21 +50,39 @@ class JiraServerProvider implements ProviderApi {
     })
   }
 
-  async getProjects() {
-    const projectsData = await this.provider?.getProjects("1")
-    const projects = projectsData?.values.map(
-      ({ key, name }: { key: string; name: string }) => ({
-        key,
-        name,
-      })
+  async getProjects(): Promise<Project[]> {
+    const response = await fetch(
+      `http://${this.requestBody.url}/rest/api/2/project?expand=lead,description`,
+      {
+        method: "GET",
+        headers: {
+          Accept: "application/json",
+          Authorization: `Basic ${Buffer.from(
+            `${this.requestBody.username}:${this.requestBody.password}`
+          ).toString("base64")}`,
+        },
```

Comment:
Why not use the getProjects() method provided by the jira client library? If we are going to make direct requests to the jira api then let's remove the library. But that comes at the cost of simplicity.

Reply:
The library presents a lot of limitations. Let's make direct api calls.